js_modules/dagster-ui/packages/ui-core/src/assets/insights/AssetCatalogInsightsLineChart.tsx

```diff
@@ -195,6 +207,41 @@ export const AssetCatalogInsightsLineChart = React.memo(
       }),
       [renderTooltipFn, rgbColors],
     );
+
+    const chartRef = useRef(null);
+    const onClick = (event: any) => {
+      if (chartRef.current) {
+        const chart = ChartJS.getChart(chartRef.current);
+        if (!chart) {
+          return;
+        }
+
+        const clickedElements = chart.getElementsAtEventForMode(
+          event,
+          'y',
+          {axis: 'x', intersect: false},
+          true,
```

Comment:
This is an issue in the Chart.js API and not your code, but boolean args are very annoying. 😆 Can you add an inline comment indicating what `true` means here?

Reply:
Good call, done!